PluginsAndFeatures/azure-toolkit-for-intellij/config/checkstyle/java.header

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Microsoft Corporation
+ * <p/>
```

Comment:
I've this question for a long time, should `<p/>` be included in copyright header?

Reply:
We shall remove <p> tag in another PR which will remove all <p> tags in all java headers.